src/DataTable/ActionDisplay.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import BulkActions from './BulkActions';
+import TableActions from './TableActions';
+import DataTableContext from './DataTableContext';
+
+// handles layout for filters, status, and bulk actions
+const ActionDisplay = () => {
+  const { bulkActions, tableActions, selectedFlatRows } = useContext(DataTableContext);
+  const noBulkActions = bulkActions?.length < 1;
+  const noTableActions = tableActions?.length < 1;
```

Comment:
The `?.` syntax here isn't preventing any errors if bulkActions were undefined. It never is because of default props in this file: https://github.com/edx/paragon/blob/5e7c3e97c24e5a7e91646540b3572d0494a5f603/src/DataTable/index.jsx#L114

That and `undefined < 1` is false as well. May be worth removing to avoid any false sense of safety.

Reply:
I mean, it could be undefined if someone were using this component on it's own? idk.